### PR TITLE
Listen for new blocks and remove user operations

### PR DESCRIPTION
**Pull request by Vid201**
Bundler now subscribes to new blocks and removes all user operations included on-chain.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -14,19 +14,21 @@ default-members = ["bin/silius"]
 
 [workspace.package]
 authors = ["Vid Kersic <vid.kersic@yahoo.com>"]
-version = "0.2.0-alpha"
+version = "0.3.0-alpha"
```
Comment (zsluedem) [inline Cargo.toml:17]: Next time, We should bump the version first and release.

**Review by zsluedem — APPROVED** [review]
LGTM

File: Cargo.toml (hunk 1 above)
Comment (Vid201) [reply Cargo.toml:17]: Yeah, that was my bad, I forgot that. Will do it next time

Comment (zsluedem) [reply Cargo.toml:17]: Not a big deal right now. Not blaming here. I made the same mistake sometimes. Additionally, we could use bot to release.

**Pull request merged**